src/App.js

```diff
@@ -35,16 +35,63 @@ const App = () => {
   //   When it is clicked on.
   //   Then pass it into the squares as a callback
 
+  const clickOnSquares = (squareId) => {
+    console.log('Is this thing working');
```

Comment:
When pushing code to be reviewed, we should eliminate any print statements or console logs

```suggestion

```